apps/web/test/lib/getSchedule.test.ts

```diff
@@ -938,15 +938,15 @@ describe("getSchedule", () => {
           {
             userId: 101,
             eventTypeId: 1,
-            startTime: `${plus2DateString}T08:30:00.000Z`,
-            endTime: `${plus2DateString}T08:29:59.999Z`,
```

Comment:
Endtime was wrong. Ended before even started. That's why no slot was blocked and it had 24 available slots